blog/_posts/btcpay-server-1-5-0.md

```diff
@@ -0,0 +1,117 @@
+date: 2022-04-28
+author: pavlenex
+category:
+  - "releases"
+tags:
+  - "btcpay-server"
+  - "release"
+  - "payouts"
+  - "dashboard"
+  - "payout processors"
+  - "update"
+coverImage: "/images/BTCPay1-5-0-desk.png"
+---
+
+After a long wait, we're finally out of our coding cave, ready to introduce a new major BTCPay Server version - 1.5.0! 
+
+This release brings in a brand-new dashboard for easier analytics and better data visualization, payout processors and schedule for later for better payout management payments, batching your payments and refunds, several UX/UI improvements, greenfield API additions and a few non-critical bug fixes.
+
+## New dashboard 📊
+
+We know that having a quick glance at your store's performance is essential. Also we understand that as a user, you would like to have an easy way to access and manage recent invoices and refunds. That's why we're introducing a **dashboard**!
+
+![](/images/1-5-0-dashboard-1.png)
+
+![](/images/1-5-0-dashboard-2.png)
+
+Dashboard visually showcases critical store's data, providing you with a way to better understand your store's performance, so that you can make more informed decisions. Furthermore, a dashboard should help with easier management of recent invoices and pending payouts and refunds.
+
+Currently, the dashboard shows data for an on-chain payment method, pending payouts and refunds, recent invoices & transactions and the data for the crowdfunding app.
+
+![](/images/1-5-0-dashboard-3.png)
+
+![](/images/1-5-0-dashboard-4.png)
+
+For more information on dashboard functionality, [check our docs](https://docs.btcpayserver.org/Dashboard/).
+
+We're building the dashboard with flexibility in mind, and the goal is extend its functionality via the plugin system. 
+
+We'd love to hear what do you think about the dashboard and how it works for you. Let us know what kind of data would you help you better manage your store.
+
+## Payout processor 🤖
+
+A payout processor is a new concept hooked into the [pull payment system], specifically the [payout part of it]. This new functionality allows you to automate the flow of the payouts.
+
+![](/images/1.5.0-payout-processor-1.png)
+
+Currently, we support payout processors for on-chain and off-chain payment methods. You can schedule payouts for on-chain, bolt11, lightning address, lnurl, etc.
+
+Set up a time interval in your store settings upon which your payouts will be batched and broadcasted to the network. 
+
+![](/images/1.5.0-payout-processor-2.png)
+
+Payout processors in a current shape and form allows you to save on transaction fees by batching the store's payments (wallet payments, refunds, payouts) and broadcasting them at a specific, pre-set time interval, for example, hourly, daily, weekly and more.
+
+### Schedule for later 📆
+
+We understand that the payout processors may sound a bit abstract, and that's why we've added a concrete use-case for it through our brand-new **schedule for later** function in the wallet.
```

Comment:
```suggestion
We understand that 'payout processors' may sound a bit abstract. This is why we've built a concrete feature that comes with it, the brand-new schedule for later option in Wallet.
```

Reply:
one sentence was better but like the quotation.